newt/downloader/downloader.go

```diff
@@ -240,17 +278,19 @@ func (gd *GithubDownloader) authenticatedCommand(path string,
 	return executeGitCommand(path, args, true)
 }
 
-func (gd *GithubDownloader) FetchFile(name string, dest string) error {
+func (gd *GithubDownloader) FetchFile(
+	path string, filename string, dstDir string) error {
+
 	var url string
 	if gd.Server != "" {
 		// Use the github API
 		url = fmt.Sprintf("https://%s/api/v3/repos/%s/%s/%s?ref=%s",
-			gd.Server, gd.User, gd.Repo, name, gd.Branch())
+			gd.Server, gd.User, gd.Repo, filename, gd.Branch())
 	} else {
 		// The public github API is ratelimited. Avoid rate limit issues with
 		// the raw endpoint.
 		url = fmt.Sprintf("https://raw.githubusercontent.com/%s/%s/%s/%s",
-			gd.User, gd.Repo, gd.Branch(), name)
+			gd.User, gd.Repo, gd.Branch(), filename)
```

Comment:
Is an http request still needed to fetch the file? With the other changes in this PR, isn't the file now already on disk? Can the GithubDownloader FetchFile be implemented in the same was as the GitDownloader FetchFile?